doc/user/layouts/partials/create-source/connector/s3/syntax.html

```diff
@@ -1,3 +1,4 @@
-*bucket_name* | The name of the bucket to scan through.
-**OBJECTS FROM SCAN BUCKET** | Materialized will scan the bucket to find the set of objects to download. See [S3 Source Details](#s3-source-details)
-*pattern* | A glob-style pattern to filter objects to ingest. See [Patterns](#patterns). Default is to ingest all files.
+**OBJECTS FROM** | Describes how to discover keys to download. See [Object discovery strategies](#object-discovery-strategies).
```

Comment:
Shouldn't all the field names be inside ``?

See question above about "strategies".  I think you also need to specify here that the options are `SCAN BUCKET`, `SQS NOTIFICATIONS`, and `MATCHING pattern`.